### ISSUE TITLE
Issue with space in PostgreSQL Full Text Search filter

### ISSUE THREAD

**Issue by MightySCollins** [tortoise/tortoise-orm#813]
**Describe the bug**
It's awesome that this library added full text search but I seem to be having a problem when I do a filter with a space.
The issue is created when using the `__search` filter on any model when the value contains a space.

**To Reproduce**
Using a basic account model the below filter will generate SQL similar to below. The search will work until a space is added.
```python
Account.filter(name__search="test name")
```
```sql
SELECT "accounts"."name" FROM "accounts" WHERE (TO_TSVECTOR("accounts"."name") @@ TO_TSQUERY('test name') OR "accounts"."name"='test name')
```

This will throw the following error as its not valid in PostgreSQL.

```
tortoise.exceptions.OperationalError: syntax error in tsquery: "test name"
```

**Expected behaviour**
I believe when a query runs it should add in either a `&` or `|` between words to make the query valid. I have tested the query with a `&` added between words and it works correctly.

**References**
https://stackoverflow.com/questions/46936417/unable-to-fetch-a-word-containing-spaces-using-to-tsvector-and-to-tsquery
https://stackoverflow.com/questions/16465466/postgres-search-query-error-if-space-used/16466057

The docs are slightly limited and I know its a new feature so just let me know if I am just using it wrong.

**Comment by long2ice** [comment]
In fact I implement full text search for MySQL and Postgres but not use it in work, what about `Account.filter(name__search="test & name")`?

**Comment by MightySCollins** [comment]
Yep that does work correctly but it probably should be handed internally or a note added to the docs to reduce confusion
